metrics/mixin/rules/rules.libsonnet

```diff
@@ -0,0 +1,11 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'ocs.rules',
+        rules: [
```

Comment:
Let's not keep it empty. 
Remove it and add when required.

Reply:
Cool